example/fly_qgc_mission/fly_qgc_mission.cpp

```diff
@@ -78,25 +78,25 @@ int main(int argc, char **argv)
     }
 
     dc.register_on_timeout([](uint64_t uuid) {
-        std::cout << "Device with UUID timed out: " << uuid << std::endl;
+        std::cout << "System with UUID timed out: " << uuid << std::endl;
         std::cout << "Exiting." << std::endl;
         exit(0);
     });
 
-    // We don't need to specify the UUID if it's only one device anyway.
+    // We don't need to specify the UUID if it's only one system anyway.
     // If there were multiple, we could specify it with:
-    // dc.device(uint64_t uuid);
-    Device &device = dc.device();
-    std::shared_ptr<Action> action = std::make_shared<Action>(device);
-    std::shared_ptr<Mission> mission = std::make_shared<Mission>(device);
-    std::shared_ptr<Telemetry> telemetry = std::make_shared<Telemetry>(device);
+    // dc.system(uint64_t uuid);
+    System &system = dc.system();
+    std::shared_ptr<Action> action = std::make_shared<Action>(system);
```

Comment:
If you plan on tidying this in future, might be good time to make these
```
auto action = std::make_shared<Action>(system);
```

Reply:
Sure I'll this time :) Thanks for reminding.